packages/http-client/src/http-client.ts

```diff
@@ -348,7 +351,7 @@ function logRequestInformation(request: HttpRequestConfig) {
  */
 export function executeHttpRequest<T extends HttpRequestConfig>(
   destination: DestinationOrFetchOptions,
-  requestConfig: T,
+  requestConfig?: T,
```

Comment:
[pp] I do not like two optional parameters in a row. I know the background that we do not want to mix our options with the ones from the http client, but now I would need a dummy object if I want to disable csrfTokenFetching which is I guess not possible because you would need to change the default method from `get` to `post`. So I guess it is fine.